src/Integration.Vsix/CFamily/CMake/CompilationDatabaseRequest.cs

```diff
@@ -50,6 +51,18 @@ public CompilationDatabaseRequest(CompilationDatabaseEntry databaseEntry, Reques
 
         public RequestContext Context { get; }
 
+        public IDictionary<string, string> EnvironmentVariables
+        {
+            get
+            {
+                return new Dictionary<string, string>
+                {
+                    // TODO - check whether a value is required. #2539
+                    { "INCLUDE", string.Empty }
```

Comment:
We'll need to work out what we need to pass, if anything, and how to calculate it.

The INCLUDE environment variable is set from the VS command line, but it is not set in VS. That either means it isn't needed for CMake builds in the IDE, or that the data is being passed in some other way. I think we'll need to discuss it with the CFamily team.